src/main/java/com/epam/preprod/sirenko/services/GetAllProductsListService.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.preprod.sirenko.services;
+
+import com.epam.preprod.sirenko.GetConsoleInput;
+import com.epam.preprod.sirenko.dao.ProductDAO;
+import com.epam.preprod.sirenko.entity.Product;
+
+import java.util.List;
+
+public class GetAllProductsListService {
```

Comment:
We already have productService for operations with products

Reply:
deleted